src/SpineOpt.jl

```diff
@@ -40,151 +41,138 @@ include("run_spineopt.jl")
 include("util/docs_utils.jl")
 include("data_structure/migration.jl")
 
-_lazy_include_file_paths = [
```

Comment:
Since we are getting rid of these we do not need to Base.invokelatest JuMP API anymore.